src/cleanlab_tlm/__init__.py

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: MIT
 from cleanlab_tlm.tlm import TLM
-from cleanlab_tlm.utils.tlm_calibrated import TLMCalibrated
+from cleanlab_tlm.utils.tlm_calibrated import TLMCalibrated, save_tlm_calibrated_state, load_tlm_calibrated_state
 from cleanlab_tlm.utils.tlm_lite import TLMLite
 
-__all__ = ["TLM", "TLMCalibrated", "TLMLite"]
+__all__ = ["TLM", "TLMCalibrated", "TLMLite", "save_tlm_calibrated_state", "load_tlm_calibrated_state"]
```

Comment:
I don't think it's necessary to have these be included here, I think if anyone wants to use it they can import it from utils and not the wildcard (people will likely discover it from the tutorials anyways so they will know what path to import it from).